### PR TITLE
For #154 Padding issue fixed and reduced repeated code

### PR DESCRIPTION
**Pull request by mohitsoni-dev**

### CODE REVIEW

**Comment by arteevraina** [comment]
@mohitsoni-dev please update your PR with some description. It can include stuff like which issue you fixed and what changes you made. Screenshots if necessary. :)

**Comment by mohitsoni-dev** [comment]
Reduced repeated code by extracting the Expanded widget and added needed padding to the text. Extracted widget:-
```
class SettingText extends StatelessWidget {
  SettingText({@required this.text});
  final String text;
  @override
  Widget build(BuildContext context) {
    return Expanded(
      flex: 3,
      child: Padding(
        padding: const EdgeInsets.all(8.0),
        child: Text(
          text,
          style: TextStyle(
            height: 1,
            fontSize: 17,
          ),
        ),
      ),
    );
  }
}
```
Before & After
before before

Please inform me if any changes are needed in this PR. Thanks.

File: lib/settings.dart [hunk 1]
```diff
@@ -112,28 +106,25 @@ class _SettingsScreenState extends State<SettingsScreen> {
...
-                              "When auto-capturing , let me adjust borders after each scan",
-                              style: TextStyle(
-                                  height: 1,
-                                  fontSize: 17
-                              ),
-                            ),
-                          ),
+                          SettingText(
+                              text:
+                                  'When auto-capturing , let me adjust borders after each scan'),
```
Comment (arteevraina) [inline lib/settings.dart]: reformat this as well.

File: lib/settings.dart [hunk 2]
```diff
@@ -148,29 +139,24 @@ class _SettingsScreenState extends State<SettingsScreen> {
...
-                            child: Text(
-                              "Run text recognition on saved pdf",
-                              style: TextStyle(
-                                  height: 1,
-                                  fontSize: 17
-                              ),
-                            ),
-                          ),
+                          SettingText(
+                              text: 'Run text recognition on saved pdf'),
```
Comment (arteevraina) [inline lib/settings.dart]: reformat this.

File: lib/settings.dart [hunk 3]
```diff
@@ -360,4 +308,25 @@ class _SettingsScreenState extends State<SettingsScreen> {
       ),
     );
   }
-}
\ No newline at end of file
+}
+
+class SettingText extends StatelessWidget {
```
Comment (arteevraina) [inline lib/settings.dart]: I would suggest adding this class in a separate file. But would like to have @anushbhatia 's view on this as well.

**Review by arteevraina — COMMENTED** [review]
Just a few changes. Other than that PR looks good. Nice Work :confetti_ball:

**Comment by mohitsoni-dev** [comment]
Understood. I'll make the suggested changes and make update the PR. Thanks.

**Comment by mohitsoni-dev** [comment]
It seems there is some problem with the lint test maybe it's outdated. @arteevraina please respond to this. Thanks.

**Comment by arteevraina** [comment]
`flutter analyze`  is failing because there are a lot of warnings in the code itself that's why it's failing. Not specifically about your code. I think this should be dealt in a separate issue. Ref: #161

**Review by arteevraina — APPROVED** [review]

**Review by smaranjitghose — APPROVED** [review]

**Pull request merged**